### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by seanlong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,27 +8,25 @@ Crosswalk-app-tools is in very early stages of development, and not suitable for
 
 ### Preparation
 
-Debian testing is the only tested platform. Node.js, the debuild tool,
-the Crosswalk runtime, and git must be functional.
+Debian and Ubuntu is the only tested platform. Node.js, the debuild tool, the Crosswalk runtime, and git must be functional.
```
Comment (rakuco) [inline README.md]: "are the only", not "is the only". Plus it would be good to mention the versions that have been verified.

File: README.md [hunk 2]
```diff
@@ -8,27 +8,25 @@ Crosswalk-app-tools is in very early stages of development, and not suitable for
...
 
 1. Download Crosswalk app tools: `git clone https://github.com/crosswalk-project/crosswalk-app-tools.git`
 2. Initialize the Crosswalk app tools: `cd crosswalk-app-tools`, then `npm install`
 3. Checkout the iOS backend: `cd node_modules`, then `git clone https://github.com/crosswalk-project/crosswalk-app-tools-deb.git crosswalk-app-tools-backend-deb`
```
Comment (rakuco) [inline README.md]: iOS looks wrong here.

Comment (seanlong) [inline README.md]: done

File: README.md (hunk 1 above)
Comment (seanlong) [inline README.md]: done.

**Comment by seanlong** [comment]
@rakuco , PTAL, thanks.

**Comment by rakuco** [comment]
lgtm, thanks

**Pull request merged**